.github/workflows/dependency-review.yml

```diff
@@ -0,0 +1,20 @@
+# Dependency Review Action
+#
+# This Action will scan dependency manifest files that change as part of a Pull Reqest, surfacing known-vulnerable versions of the packages declared or updated in the PR. Once installed, if the workflow run is marked as required, PRs introducing known-vulnerable packages will be blocked from merging.
+#
+# Source repository: https://github.com/actions/dependency-review-action
+# Public documentation: https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-dependency-review#dependency-review-enforcement
+name: 'Dependency Review'
+on: [pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@v3
+      - name: 'Dependency Review'
+        uses: actions/dependency-review-action@v1
```

Comment:
For both "uses" lines: we're trying to use hashes for actions instead of tags. This is because tags are mutable and in practice GitHub does move the tags so our workflows end up changing without us knowing that they change. 

This might not be that important for this workflow (since it's read permissions only) but for consistency, could you update this commit so that it refers to the specific commit hashes instead of tags? There are examples in our other workflows.

Reply:
I have updated them to SHA. PTAL. Thanks